deploy_tasks/sm_graphql.yml

```diff
@@ -23,6 +26,12 @@
     chdir: "{{ sm_graphql_home }}"
     executable: /bin/bash
 
+- name: Dereferencing metadata schema
+  shell: node ./metadata/deref_schema.js > metadata_schema.json
```

Comment:
@Ren22 Metadata repository has no branch with deref_schema.js script

Reply:
@intsco True, it was only in my fork, created a pull-request to the submodule repo.